lib/pbench/cli/agent/commands/tools/clear.py

```diff
@@ -24,7 +22,7 @@ class ClearTools(ToolCommand):
     def __init__(self, context):
         super().__init__(context)
 
-    def execute(self):
+    def execute(self) -> int:
```

Comment:
Is this "change for change sake?" Asking for a friend... 😆

Reply:
No...I added a `return` statement (and changed the basis for another), and I wasn't sure whether it should return an `int` or a `bool`, so I figured I should record my conclusion.

(Tell your friend, "It's OK."  😉)